test-end-to-end/android-maestro/e2e-tests/003_006_course_view_and_modules_can_expand_collapse.yaml

```diff
@@ -0,0 +1,51 @@
+appId: com.toughra.ustadmobile
+env:
+  TESTRESULTSDIR: "${TESTRESULTSDIR}/003_006_course_view_and_modules_can_expand_collapse"
+  TESTNAME: "003_006_course_view_and_modules_can_expand_collapse"
+---
+- clearState: com.toughra.ustadmobile
+- runFlow:
+    file: "../common/startserver.yaml"
+    env:
+      CONTROLSERVER: ${CONTROLSERVER}
+      TESTSERIAL: ${TESTSERIAL}
+      TESTNAME: ${TESTNAME}
+- runFlow: "../common/clear_state_and_get_started.yaml"
+- runFlow: "../common/select_site.yaml"
+- runFlow:
+    file: "../common/user_login.yaml"
+- runFlow:
+    file: "../common/add_course.yaml"
+    env:
+      TESTCOURSENAME: "Course view Test"
+- runFlow:
+    file: "../common/add_new_student_inside_course.yaml"
+    env:
+      TESTCOURSENAME: "Course view Test"
+      STUDENTFIRSTNAME: "Muhammed"
+      STUDENTLASTNAME: "Khan"
+      GENDER: "Male"
+      PASSWORD: "test"
+- back
+- back
+- tapOn: "OVERVIEW"
+- runFlow:
+    file: "../common/add_demo_block_structure.yaml"
+    env:
+      MODULETITLE: "SEMESTER 1"
+      TEXTTITLE: "PRIMARY COLOURS"
+      TEXT: "RED,BLUE,GREEN"
+- runFlow: "../common/clear_state_and_get_started.yaml"
+- runFlow: "../common/select_site.yaml"
+- runFlow:
+    file: "../common/user_login.yaml"
+    env:
+      USERNAME: "Muhammed"
+      PASSWORD: "test"
+- tapOn: "courses"
+- tapOn: "Course view Test"
+- tapOn: "SEMESTER 1"
+- takeScreenshot: "${TESTRESULTSDIR}/able_to_collapse_module"
```

Comment:
Needs assertions that the module is collapsed / expanded. There must be a submodule (e.g. simple text module), which should be visible when expanded, invisible when collapsed.